### ISSUE TITLE
Initial set of Cohere+ exemplars

### ISSUE THREAD

**Issue by catherinet1** [life-itself/ecosystem#186]
## Job stories

When presenting project to others and/or discussing metadata i want an initial set of Cohere+ exemplar profiles so that we can use them as a basis for discussion

Plus some (informal) written notes (could be a discussion thread) about where we are at ...
- What is the "center(s)" we care about and the related dimensions (e.g. PIE)
- What are examples of things that closer / further on important dimensions (for didactic purposes)

## Acceptance

- [x] 3-5 exemplars **3 refined out of 6 identified as possible exemplars: [42 Acres](https://ecosystem.lifeitself.org/tbd/42-Acres); [Joysong Collective](https://ecosystem.lifeitself.org/tbd/Joysong%20Collective); and [Center for Systems Awareness](https://ecosystem.lifeitself.org/tbd/Center%20for%20Systems%20Awareness)**
- [x] Discussion thread on the written notes **Here: https://github.com/orgs/life-itself/discussions/972. Some more reflection could be added, e.g. have picked out some common threads but not key differences. We also discussed in 2023-12-20 meeting (and recorded).**
- [ ] [Bonus] Nice to have: write up discussion thread into blog post **Let's do this after #204**
- [x] Principles of our current mapping approach (see notes in Analysis below) are documented at ecosystem.lifeitself.org/sensemaking

## Tasks 

- [x] Move exemplars from cohereplus repo into ecosystem repo **Created as new notes in tbd folder**
  - [x] https://github.com/life-itself/cohereplus/blob/main/profiles/awakened-%20life-project.md
  - [x] https://github.com/life-itself/cohereplus/blob/main/profiles/42-acres.md
  - [x] https://github.com/life-itself/cohereplus/blob/main/profiles/joysong-collective.md
  - [x] https://github.com/life-itself/cohereplus/blob/main/profiles/embercombe.md
  - [x] https://github.com/life-itself/cohereplus/blob/main/profiles/the-alternative.md
- [x] Push additions
- [x] Delete the above files from cohereplus repo
- [x] Edit profiles to become final exemplars **Did 3: 42 Acres, Joysong Collective, and Center for Systems Awareness**
  - [x] 42 Acres **https://ecosystem.lifeitself.org/tbd/42-Acres**
    - [x] Fix build **Tried renaming file from 42 Acres.md to 42-Acres.md**
  - [ ] Awakened Life Project **don't prioritise this one as it seems to have been archived and taken a new name -- "The Fire of the Heart"**
  - [x] Joysong Collective **https://ecosystem.lifeitself.org/tbd/Joysong%20Collective**
  - [ ] Embercombe
  - [ ] The Alternative
  - [x] Center for Systems Awareness **https://ecosystem.lifeitself.org/tbd/Center%20for%20Systems%20Awareness**
- [x] Link to Exemplars on Index page **Added: 42 Acres, Joysong, Center for Systems Awareness**
- [ ] Integrate and delete (?) profiles/42-acres **Leave for now**
- [ ] Integrate and delete (?) profiles/embercombe **Leave for now**
- [ ] Integrate and delete (?) profiles/the-alternative-uk **Leave for now**
- [x] Discussion thread **Here: https://github.com/orgs/life-itself/discussions/972**
  - [x] Share in research community group chat
~~- [ ] Blog post **See note in Acceptance**~~
  ~~- [ ] Share in research community group chat~~
- [ ] (Re?)open issue on profile template / specs **Doesn't seem necessary given focus for now on "wiki over database"**

### Principles

- wiki over database
  - I don't really care that much about fields right now ... (sure later)
  - use links for categorization if you want (a la wikipedia)
- focus on quality of profiles in terms of summary and drawing out the things interesting to us ...
  - e.g. their philosophy/theory of change
  - e.g. what they *actually* do
  - e.g. "impact" or potential impact - whether they are "getting stuff done" and doing so in sustainable way (?)
- then do discussion to clarify things ...
  - e.g. what tribes are emerging
  - what features do we care about ...

**Comment by catherinet1** [comment]
FIXED. Identified 6 possible exemplars and refined 3 (see Acceptance for links to those profiles) and added these to tbd/Index.md. Discussion thread posted (link in Acceptance) and principles of current mapping approach added to ecosystem.lifeitself.org/sensemaking.
